js/components/TodoList.js

```diff
@@ -0,0 +1,7 @@
+import TodoItem from "./TodoItem.js";
+
+const TodoList = ({list}) => `
+	${list.map(todo => TodoItem(todo)).join('')}
+`;
```

Comment:
```suggestion
const TodoList = ({ list }) => list.map(TodoItem).join('');
```
굳이 템플릿리터럴을 사용하지 않아도 될 것 같습니다.